packages/flutter/lib/src/painting/_network_image_io.dart

```diff
@@ -122,3 +181,81 @@ class NetworkImage extends image_provider.ImageProvider<image_provider.NetworkIm
   @override
   String toString() => '$runtimeType("$url", scale: $scale)';
 }
+
+class _PendingLoadRequest {
+  _PendingLoadRequest(this.sendRequest, this.handleError);
+
+  Function sendRequest;
+  Function handleError;
+}
+
+
+class _DownloadRequest {
+  _DownloadRequest(this.sendPort, this.uri, this.headers, this.debugNetworkImageHttpClientProvider);
+
+  final SendPort sendPort;
+  final Uri uri;
+  final Map<String, String> headers;
+  final HttpClientProvider debugNetworkImageHttpClientProvider;
+}
+
+class _HttpClientIsolateParameters {
+  _HttpClientIsolateParameters(this.sendPort);
+
+  final SendPort sendPort;
+}
+
+// We set `autoUncompress` to false to ensure that we can trust the value of
+// the `Content-Length` HTTP header. We automatically uncompress the content
+// in our call to [consolidateHttpClientResponseBytes].
+final HttpClient _sharedHttpClient = HttpClient()..autoUncompress = false;
+const Duration _idleDuration = Duration(seconds: 60);
+
+void _handleHttpClientGet(_HttpClientIsolateParameters params) {
```

Comment:
`_handleHttpClientGet` makes it sound like it's issuing an HTTP GET request.  How about calling it something like `_initializeWorkerIsolate()` instead?

Reply:
Done